### PR TITLE
【translate】mds/topology

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/mds/topology/topology.h [hunk 1]
```diff
@@ -90,75 +90,79 @@ class Topology {
...
      *
-     * @return 错误码
+     * @return error code
      */
     virtual int UpdateChunkServerStartUpTime(uint64_t time,
                          ChunkServerIdType id) = 0;
 
     /**
-     * @brief 更新copyset 拓扑信息
+     * @brief update copyset topology info
```
Comment (xu-chaojie) [inline src/mds/topology/topology.h:156]: topology 这个词不要，copyset info就够了，容易引起误解

File: src/mds/topology/topology.h [hunk 2]
```diff
@@ -517,23 +521,23 @@ class TopologyImpl : public Topology {
             return true;}) const override;
 
     /**
-     * @brief 获取chunksever的所属physicalPool Id
+     * @brief get physicalPool Id that the chunksever belongs to
```
Comment (xu-chaojie) [inline src/mds/topology/topology.h:524]: chunkserver

File: src/mds/topology/topology_chunk_allocator.cpp [hunk 3]
```diff
@@ -185,9 +188,9 @@ bool TopologyChunkAllocatorImpl::ChooseSingleLogicalPool(
                   << ", diskAlloc:" << alloc
                   << ", diskRemainning:" << diskRemainning
                   << "}";
-
+        // choose logical pool according to its weight
```
Comment (xu-chaojie) [inline src/mds/topology/topology_chunk_allocator.cpp:191]: 这句翻译不对

Comment (xu-chaojie) [inline src/mds/topology/topology_chunk_allocator.cpp:191]: 这句翻译不对

File: src/mds/topology/topology.h (hunk 1 above)
Comment (SeanHai) [reply src/mds/topology/topology.h:156]: > topology 这个词不要，copyset info就够了，容易引起误解

done

File: src/mds/topology/topology.h (hunk 2 above)
Comment (SeanHai) [reply src/mds/topology/topology.h:524]: > chunkserver

done

File: src/mds/topology/topology_chunk_allocator.cpp (hunk 3 above)
Comment (SeanHai) [reply src/mds/topology/topology_chunk_allocator.cpp:191]: > 这句翻译不对

这块没有对应的中文，那这里的if条件是什么意思？

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**